src/CodeGen_PTX_Dev.cpp

```diff
@@ -449,8 +449,9 @@ vector<char> CodeGen_PTX_Dev::compile_to_src() {
     PassManagerBuilder b;
     b.OptLevel = 3;
     b.Inliner = createFunctionInliningPass(b.OptLevel, 0, false);
-    b.LoopVectorize = true;
+    b.LoopVectorize = !target.has_feature(Target::DisableLLVMLoopOpt);
```

Comment:
The logic used in Codegen_LLVM.cpp also uses the EnableLLVMLoopOpt feature; we should probably mimic that logic here too, for consistency.

Reply:
(I took the liberty of pushing a change that added that logic. PTAL)

Reply:
Thanks